docs/developers/get_started/building_with_bazel_linux.md

```diff
@@ -59,7 +59,7 @@ $ python3 configure_bazel.py
 Run all core tests:
 
 ```shell
-$ bazel test -k iree/...
+$ bazel test -k compiler/... runtime/...
 ```
```

Comment:
This script https://github.com/openxla/iree/blob/main/build_tools/bazel/build_core.sh is what our CI runs. That used to include directories, but the non-exhaustive list missed some paths. As of https://github.com/openxla/iree/pull/9206, you should just be able to `bazel test -k //...`

Reply:
Yep, added. I just had to get my cuda environment set up correctly. This works at least to build now and the failing tests are probably mostly due to not having the right gpu

Reply:
:+1: cool, this should be ready to merge once the other places are updated too: https://github.com/openxla/iree/tree/main/docs/developers/get_started (can just search the repo for `bazel test -k iree/...`)